emoji/emoji_transfer_service.py

```diff
@@ -12,12 +22,8 @@ def transfer(source_client, destination_client, source_emoji_name):
     source_dict = source_client.emoji_dict()
     destination_dict = destination_client.emoji_dict()
 
-    emoji_path = source_dict.get(source_emoji_name)
+    emoji_path = url-url_even_if_alias(source_dict, source_emoji_name)
```

Comment:
is this `url-url` bit a typo or a python thing?

Reply:
typo, fix is up now